libnavigation-base/src/main/java/com/mapbox/navigation/base/route/RouteUrl.kt

```diff
@@ -34,6 +34,12 @@ class RouteUrl(
         private const val QUERY_PARAM_ROUNDABOUT_EXITS = "roundabout_exits"
         private const val QUERY_PARAM_ENABLE_REFRESH = "enable_refresh"
 
+        const val STEPS_DEFAULT_VALUE = false
+        const val VOICE_INSTRUCTION_DEFAULT_VALUE = false
+        const val BANNER_INSTRUCTION_DEFAULT_VALUE = false
+        const val ROUNDABOUT_EXITS_DEFAULT_VALUE = false
+        const val ENABLE_REFRESH_DEFAULT_VALUE = true
```

Comment:
Should this be true?

/cc @Guardiola31337

Reply:
Currently we're enabling refresh by default 👀 https://github.com/mapbox/mapbox-navigation-android/blob/c3d1f4cebb2d57a71bfbb6c4bf8953e9da7eca98/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationRoute.kt#L742 but we can definitely discuss if it makes sense now with the new setup.

Reply:
The same thing happens with steps, voice and banner instructions and roundabout exits 👀 https://github.com/mapbox/mapbox-navigation-android/blob/c3d1f4cebb2d57a71bfbb6c4bf8953e9da7eca98/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationRoute.kt#L735

Reply:
Also want to note that the Navigation use case is different from the Directions one i.e. the defaults could be different (that's why we had some of them unmatched and always default to what makes most sense in Navigation).

Also, it's difficult to know what's the default value from the name e.g. `ENABLE_REFRESH_DEFAULT_VALUE` and it's hard to tell when reading the code. For understandability, should we rename it into something more explicit?

Reply:
@Guardiola31337 you mean like `ENABLE_REFRESH_DEFAULT_VALUE ` -> `ENABLE_REFRESH_DEFAULT_VALUE_FALSE`

Reply:
I'd even remove the `DEFAULT` so something like this `REFRESH_ENABLED` / `REFRESH_DISABLED` (in this particular case I'd also remove the `ENABLE` at the beginning). What do you think? I think it reads 👌

Reply:
I think it's intension to hide value under the name of value 😃 
suggest next naming: `REFRESH_INITIAL = false/true`

Reply:
We'd be in the same situation, we wouldn't know when reading the code. BTW what's the purpose of hiding the value? It's an internal default value so the more context the better, for readability and understandability.

Reply:
if we'd like to change this value we should change name of it as well, it's not the best approach. btw it's already public, not hidden, value. 
To see value you need to clic on it, open docs or class where this value is, but not ready VALUE from name, it's strange at least, intread we could just write value directly, without saving to property/field

Reply:
All of these constants should be `internal`, don't see any value of exposing them publicly, as they're only used internally for handling the default values. You can look at the current implementation and they're actually `internal` e.g. https://github.com/mapbox/mapbox-navigation-android/blob/f6963eeff0b84c0846026bb833074badc06fa988/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationConstants.kt#L140

Reply:
they could be internal, I'm agree

Reply:
these fields are used in different module, so cannot be internal (are used in `MapboxOnboardRouter`)

Reply:
@RingerJK :eyes: https://github.com/mapbox/mapbox-navigation-android/issues/2417

Reply:
@Guardiola31337 got it, thanks. Need sync. I think it works good for whole class, but if we should hide `field` only, it's not working so pure

Reply:
Not sure if I follow / what you mean by `it's not working so pure` but yeah let's sync up internally :+1:

Reply:
btw for `RouteUrl` it works

Reply:
What does it work for `RouteUrl`? :thinking:

Reply:
@Guardiola31337 [PR](https://github.com/mapbox/mapbox-navigation-android/pull/2426)